Controller/message.js

```diff
@@ -0,0 +1,27 @@
+import messageModel from "../Model/message.js";
+
+export const newMessage = async (req, res) => {
+  const { message_data, message_type, sender_id, conversation_id } = req.body;
+  const newMessage = new messageModel({
+    message_data,
+    message_type,
+    sender_id,
+    conversation_id,
+  });
+  try {
+    const result = await newMessage.save();
+    res.status(200).json({ message: "message created successfully" });
+  } catch (error) {
```

Comment:
It's a good practice to provide a more descriptive error message, such as "Internal Server Error." Also, it's beneficial to log the error for debugging purposes.